tutorials/08-voting/contracts/Voting.cdc

```diff
@@ -0,0 +1,132 @@
+/*
+*
+*   In this example, we want to create a simple voting contract
+*   where a polling place issues ballots to addresses.
+*
+*   The run a vote, the Admin deploys the smart contract,
+*   then initializes the proposals.
+*   The array of proposals cannot be modified after it has been initialized.
+*
+*   Users can create ballots and vote only with their GovernanceToken balance prior to when
+*   proposal was created.
+*
+*   Every user with a ballot is allowed to approve their chosen proposals.
+*   A user can choose their votes and cast them
+*   with the tx_05_SelectAndCastVotes.cdc transaction.
+*
+*/
+
+import GovernanceToken from "./GovernanceToken.cdc"
+
+pub contract Voting {
```

Comment:
Can you make this a more specific name? like `CadenceExampleVotingTutorial`

Reply:
So far, in all other tutorials, only path names contained a reference to 'Cadence' and 'Tutorial'. Are you sure we should rename the contracts?